packages/react-ui-core/src/Gmap/index.js

```diff
@@ -1,3 +1,5 @@
 export { default } from './Gmap'
 export { default as Gmap } from './Gmap'
 export { default as GmapSpinner } from './GmapSpinner'
+export { default as Marker } from './Marker'
+export { default as GmapInteraction } from './GmapInteraction'
```

Comment:
```
export { GmapInteraction } from './GmapInteraction'
```
Since it's not export default. Either that, or make it a default export

Reply:
Good catch!